docs/docs/installation/index.md

```diff
@@ -121,6 +121,20 @@ Some helpful hints:
 
     The output shows all Keptn components that are running on your cluster.
 
+### Alternative installation methods
+
+#### Install Keptn via Glasskube
+
+Glasskube is a package manager for Keptn that simplifies the installation process and can be used to automatically upgrade Keptn to the latest version. For more information how to install Glasskube, see [Glasskube Installation](https://glasskube.dev/docs/getting-started/install/).
+
+To install Keptn via Glasskube after Glasskube is installed, run the following command:
+
+``` shell
+glasskube install keptn
```

Comment:
Looks nice, thank you! One question: is it expected that glasskube does not install the latest version of Keptn? It installs keptn-v0.10.0 (released Feb 8). Since that we already release two v2.0.0-rc versions

Reply:
Hi @odubajDT we just have been able to follow up with most tasks after KubeCon, see:

https://github.com/glasskube/packages/pull/59